src/server/templates/model.html

```diff
@@ -61,56 +62,61 @@ <h5>Global mitigation strength</h5>
       <div class="row mitigation-strenght">
         <div class="col-sm-auto col-xm-12">
           <label class="btn btn-secondary active beta-0">
-            <input
-              type="radio"
-              name="mitigation-none"
-              id="mitigation-none"
-              autocomplete="off"
-            />
+            <input type="radio" name="mitigation-none" id="mitigation-none" autocomplete="off" />
             None
           </label>
         </div>
         <div class="col-sm-auto col-xm-12">
           <label class="btn btn-secondary active beta-05">
-            <input
-              type="radio"
-              name="mitigation-weak"
-              id="mitigation-weak"
-              autocomplete="off"
-            />
+            <input type="radio" name="mitigation-weak" id="mitigation-weak" autocomplete="off" />
             Weak
           </label>
         </div>
         <div class="col-sm-auto col-xm-12">
           <label class="btn btn-secondary active beta-04">
-            <input
-              type="radio"
-              name="mitigation-moderate"
-              id="mitigation-moderate"
-              autocomplete="off"
-            />
+            <input type="radio" name="mitigation-moderate" id="mitigation-moderate" autocomplete="off" />
             Moderate
           </label>
         </div>
         <div class="col-sm-auto col-xm-12">
           <label class="btn btn-secondary active beta-03">
-            <input
-              type="radio"
-              name="mitigation-strong"
-              id="mitigation-strong"
-              autocomplete="off"
-            />
+            <input type="radio" name="mitigation-strong" id="mitigation-strong" autocomplete="off" />
             Strong
           </label>
         </div>
       </div>
-      {{message | default("Different global mitigation practices will change the
-      rate of spread of COVID-19")}}
       <hr />
       <!-- Create a div where the graph will take place -->
       <div id="my_dataviz" class="svg-container"></div>
+
+      <p>
+        The impact on [...country...] is simulated using a global, real-time database of human travel patterns.
```

Comment:
```suggestion
        The impact on <span id="country-graph"></span> is simulated using a global, real-time database of human travel patterns.
```

Reply:
It will have to be updated via js in lines.js

Reply:
if/when you complete it, please refer to this issues: https://github.com/epidemics/covid/issues/139

Reply:
@lagerros It looks to me like #139 is just a reference to this thread. Is that correct? It's currently assigned to me. Should I wait for this to get merged into staging or should I pull down your branch and update this PR myself?

Reply:
Oh sorry I solved #139 there already :(